main.nf

```diff
@@ -780,6 +781,30 @@ process idscoreswitcher3 {
      """
 }
 
+process luciphor {
+
+    publishDir "${params.outdir}/logs", mode: 'copy', pattern: '*.log'
+
+    input:
+     file id_file from id_files_idx_feat_perc_fdr_filter_switched.mix(id_files_idx_ForIDPEP_fdr_switch_idpep_switch_filter_switch)
```

Comment:
please introduce the additional settings as well (at least tolerances and fragment method). best: all non-advanced.

Reply:
for tolerances and everything that is present in an SDRF please use a settings map as for the other tools. see e.g. line 183